static/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -186,6 +189,29 @@ const QuickTraceContainer = styled('div')`
   }
 `;
 
+function EventID({event}: {event: Event}) {
+  return (
+    <EventIDContainer>
+      <EventIDWrapper>{getShortEventId(event.eventID)}</EventIDWrapper>
+      <Tooltip title={event.eventID} position="top">
+        <Clipboard value={event.eventID}>
+          <IconCopy color="subText" />
+        </Clipboard>
+      </Tooltip>
+    </EventIDContainer>
+  );
+}
+
+const EventIDContainer = styled('div')`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
```

Comment:
Does cursor pointer make sense for all of it since only the clipboard should be "clickable"?